playbook/app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.tsx

```diff
@@ -36,59 +36,36 @@ const dateTimeIso = (dateValue: Date) => {
 
 const DateRangeInline = (props: DateRangeInlineProps): React.ReactElement => {
   const {
-    icon = false,
-    dark = false,
-    size = "sm",
     align = "left",
+    className,
+    dark = false,
     data = {},
+    endDate,
     htmlOptions = {},
+    icon = false,
+    size = "sm",
     startDate,
-    endDate,
-    className,
   } = props;
 
-  const iconContent = () => {
```

Comment:
Hey @anthonymig88 quick question: Any particular reason we're removing the function here and moving this extraction back into the return? Extraction is always better for performance and speed!

cc: @nickamantia @jasperfurniss

Reply:
@nidaqg I made the change to simplify the code. I agree that extraction is generally better for performance yet in this specific case, I felt the difference is negligible from what I've been learning. If performance becomes a concern with my change, I'm open to revisiting the changes.